src/main/resources/db/changelog/v0.1/04.13.2020-Add_Appointment_Schema.sql

```diff
@@ -0,0 +1,42 @@
+--liquibase formatted sql
+--changeset hlebstraltsou:04.14.2020-Calendar_Schema.sql
+CREATE TABLE appointment (
+  appointment_id BINARY(16) NOT NULL,
+  `subject` VARCHAR(256) NOT NULL,
+  start_timestamp DATETIME NOT NULL,
+  end_timestamp DATETIME NOT NULL,
+  location VARCHAR(256),
+  description VARCHAR(1256),
+  video_room_id BINARY(16),
+  is_all_day BOOLEAN DEFAULT 0,
+  recurrence_rule VARCHAR(512),
+  owner_id BINARY(16) NOT NULL,
```

Comment:
I gues if you decide to use User as n ownerId I believe you\d better add a forerign key for that. Thanks